src/applications/yellow-ribbon/actions/index.js

```diff
@@ -104,8 +104,8 @@ export const fetchResultsThunk = (options = {}) => async dispatch => {
           contributionAmount: contributionAmount || undefined,
           numberOfStudents: numberOfStudents || undefined,
         },
-        'search-total-results': response.results.length,
-        'search-total-result-pages': Math.ceil(
+        'search-results-total-count': response.results.length,
+        'search-results-total-page': Math.ceil(
```

Comment:
Did you let analytics know about these key changes? I'm guessing this is coming from their request anyways but just double checking! 🙂

Reply:
Yep! It's an effort to make all va gov search's have the same skeleton of properties.